charts/signoz/templates/otel-collector/deployment.yaml

```diff
@@ -63,7 +63,7 @@ spec:
           imagePullPolicy: {{ .Values.otelCollector.image.pullPolicy }}
           args: 
           - "job"
-          - "{{ include "schemaMigrator.fullname" . }}-{{ .Release.Revision }}"
+          - "{{ include "schemaMigrator.fullname" . }}-{{ include "schemaMigrator.jobSuffix" . }}"
```

Comment:
@dhawal1248 I suppose you need to do the same for the init-container in Otel-Collector-Metrics as well.